create-quasar/utils/index.js

```diff
@@ -218,6 +218,54 @@ module.exports.ensureOutsideProject = function () {
   }
 }
 
+function hasGit() {
+  try {
+    exec("git --version");
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
+
+function hasProjectGit(cwd) {
+  try {
+    exec("git status", { stdio: "ignore", cwd });
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
+
+module.exports.initializeGit = async function (cwd) {
+  if (!hasGit()) {
+    logger.warning(
+      "Git is not present on the system, skipping repo initialization..."
+    );
+    return;
+  }
+
+  if (hasProjectGit(cwd)) {
+    logger.warning(
+      "The project already have an initialized Git repository, skipping repo initialization..."
+    );
+    return;
+  }
+
+  await exec("git init", { cwd });
+  await exec("git add -A", { cwd });
+
+  try {
+    await exec(
+      "git commit -m 'Initialize the project 🚀' --no-verify",
```

Comment:
I think there should be a change to `git commit -m 'chore: Initialize the project 🚀`, because 'Conventional Commit' is popular.